assets/html/scss/juliamono.scss

```diff
@@ -0,0 +1,44 @@
+$juliamono-base-url: "https://cdnjs.cloudflare.com/ajax/libs/juliamono/0.037";
+
+@font-face {
+    font-family: JuliaMono;
+    font-weight: 300;
+    font-style: normal;
+    src: local('JuliaMono'), url("#{$juliamono-base-url}/JuliaMono-Light.woff2") format("woff2");
```

Comment:
What exactly do these different versions do? For my own website I used the one called `Regular` but that one is not even included here.

Reply:
I've not yet found out how CSS font weights works. :) 

In the font file I have these definitions:

Light 300

Regular 400

Medium 500

Semibold 600

Bold 700

Extrabold 800

Black 900

- Regular is the one I'd use by default.

Reply:
Okay. What determines which one is used? Will all those file be loaded by the browser? And for the case of Documenter, are all of them used?

Reply:
In my Documenter docs CSS I use just this one:

```
@font-face {
	font-family: JuliaMono-Regular;
	font-display: fallback;	/* allow 100 milliseconds then display fallback */
	src: url("https://cdn.jsdelivr.net/gh/cormullion/juliamono/webfonts/JuliaMono-Regular.woff2");
}
```

Probably not a good idea to ask for them all :)

Reply:
For what it's worth, it looks like Chrome at least is smart enough not to download the `.woff2` if you're not actually using it. Currently, with this setup, on the `-Medium` is actually loaded, because the code blocks have the (default) weight of `400`.

That said, I agree that, if we're gonna have the CSS file here, we should only have `-Regular` there. However, @cormullion, what is the actual difference between the `.woff2` files of the different weights? Why are you generating different ones in the first place?

Reply:
Each `woff2` file contains the glyph shapes and data for one specific weight, so you need a separate file for each weight. (I think variable fonts might come in one big file, but this isn’t one of those.)